src/store/prompts/prompts.reducers.ts

```diff
@@ -219,9 +225,48 @@ export const promptsSlice = createSlice({
 
       state.folders = state.folders.concat(newFolder);
     },
+    createTemporaryFolder: (
+      state,
+      {
+        payload,
+      }: PayloadAction<{
+        relativePath?: string;
+      }>,
+    ) => {
+      const folderName = getAvailableNameOnSameFolderLevel(
+        [
+          ...state.temporaryFolders,
+          ...state.folders.filter((folder) => folder.publishedWithMe),
```

Comment:
`publishedWithMe` is used only for root items now. Maybe it should be fixed and mark all items